webdriver-spec.html

```diff
@@ -2882,6 +2888,68 @@
  whose <a><code>readOnly</code> attribute</a> is set.
 
 <section>
+<h3>Element Interactibility</h3>
+
+<p>In order to determine if an element can be interacted with,
+ WebDriver performs hit-testing to find
+ if the interaction will be able to reach the requested element.
+
+<p>An <dfn>interactible element</dfn> is defined to be
+ the first non-transparent <a>element</a>,
```

Comment:
This is just the non-normative description, but it means `opacity` set to != "1".